pop-api/integration-tests/src/fungibles/mod.rs

```diff
@@ -535,10 +535,10 @@ fn burn_works() {
 		let amount: Balance = 100 * UNIT;
 
 		// Token does not exist.
-		assert_eq!(burn(&addr, 1, &BOB, amount), Err(Module { index: 52, error: [3, 0] }));
+		assert_eq!(burn(&addr, 1, &BOB, 0), Err(Module { index: 52, error: [3, 0] }));
 		let token = assets::create(&ALICE, 1, 1);
-		// Bob has no tokens and therefore doesn't exist.
-		assert_eq!(burn(&addr, token, &BOB, 1), Err(Module { index: 52, error: [1, 0] }));
+		// Bob has no tokens and therefore doesn't have any balance.
```

Comment:
Please read the sentence again if you change something, this sentence doesn't make sense.

Reply:
// Bob has no tokens

Reply:
Agree...